### PR TITLE
Production v2.7

### PR DESCRIPTION
**Pull request by Mschuddings**

### CODE REVIEW

File: lib/themes/erasmus_30year_anniversary/assets/sass/layout/_discover.scss [hunk 1]
```diff
@@ -61,6 +60,20 @@
...
+@include media($tablet-min-width) {
+  .discover-erasmus-plus {
+    header {
+      margin: 5em 0 0 40px;
+      margin-top: 5em;
+      max-width: calc(50% - 40px);
+    }
+  }
+}
+
```
Comment (jonhy81) [inline lib/themes/erasmus_30year_anniversary/assets/sass/layout/_discover.scss]: Please keep only one line, remove multiple empty lines.

File: lib/themes/erasmus_30year_anniversary/assets/sass/layout/_discover.scss [hunk 2]
```diff
@@ -93,5 +104,25 @@
...
 @include media($largedesktop-min-width) {
+  .discover-erasmus-plus {
+    header {
+      padding: $padding-medium;
+      margin-left: $padding-large;
+      background: rgba(255,255,255, .7);
+    }
+  }
+}
+
```
Comment (jonhy81) [inline lib/themes/erasmus_30year_anniversary/assets/sass/layout/_discover.scss]: The same here, remove multiple lines.

Comment (Mschuddings) [inline lib/themes/erasmus_30year_anniversary/assets/sass/layout/_discover.scss]: Resolved

File: lib/themes/erasmus_30year_anniversary/assets/sass/layout/_discover.scss (hunk 1 above)
Comment (Mschuddings) [inline lib/themes/erasmus_30year_anniversary/assets/sass/layout/_discover.scss]: Dear,

This has been Resolved

**Pull request merged**